content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -896,7 +896,7 @@ const user = await prisma.user.update({
 ```
 
 ### <inlinecode>upsert()</inlinecode>
-
+*This section covers the usage of the general upsert() operation. To learn about using nested upsert queries within update(), reference the documentation [here](#upsert-1).*
```

Comment:
```suggestion

*This section covers the usage of the general `upsert()` operation. To learn about using nested upsert queries within `update()`, reference the documentation [here](#upsert-1).*

```

Reply:
Can we find a better link text than "here"? I think the docs have a styleguide that includes not using "here" and similar (or had in the past).

Reply:
```suggestion
*This section covers the usage of the general `upsert()` operation. To learn about using [nested upsert queries](#upsert-1) within `update()`, reference the linked documentation.*
```

Ya, I followed the same format as the other. Does that work?

Reply:
Definitely better for me, your colleagues hopefully know the _actual_ style guide and if that is ok with it.

Reply:
```suggestion

<Admonition type="info">

This section covers the usage of the `upsert()` operation. To learn about using [nested upsert queries](#upsert-1) within `update()`, reference the linked documentation.

</Admonition>
```